runtime/src/accounts_background_service.rs

```diff
@@ -184,6 +184,12 @@ impl SnapshotRequestHandler {
                     status_cache_slot_deltas,
                 } = snapshot_request;
 
+                if !snapshot_root_bank.is_validator_initialized() {
+                    // this cannot be trusted yet
+                    // may need a different return value here
+                    return Ok(0);
```

Comment:
@brooksprumo I'd like to not create a snapshot for this slot. perhaps I should return a new enum that says 'skipped'. Or, does 0 do something useful? I considered cutting offer the senders, but there are currently 2 of them. That seems more brittle.

Reply:
@mvines this will prevent us from calculating accounts hash, which should prevent us from knowing or publishing it.

Reply:
Another options is to not send this bank to the SnapshotRequestHandler during `bank_forks::set_root()`. That's  more clear IMO.